michiganelections_api/src/main/java/io/michiganelections/api/Api.kt

```diff
@@ -0,0 +1,42 @@
+package io.michiganelections.api
+
+import retrofit2.Retrofit
+import retrofit2.http.GET
+import retrofit2.http.Query
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+object Api {
+  interface Endpoint {
+    @GET("elections")
+    suspend fun elections(
+      @Query("active") isActive: Boolean = true
+    ): ResultPage<Election>
+
+    @GET("registrations")
+    suspend fun registrations(
+      @Query("first_name") firstName: String,
+      @Query("last_name") lastName: String,
+      @Query("birth_date") birthDate: String,
+      @Query("zip_code") zipcode: String
+    ): VoterRegistration
+  }
+
+  class Service(retrofit: Retrofit) {
+    private val endpoint = retrofit.create(Endpoint::class.java)
+
+    suspend fun getElections(): ResultPage<Election> {
```

Comment:
Thoughts on unit tests?

Reply:
I don't think there is any testable material here yet, i was just playing around with the API and wrote some data classes to represent the json objects.